tests/perf/mt_ubench_spdlog.cpp

```diff
@@ -1,31 +1,67 @@
+/**
+ * *****************************************************************************
+ * \file perf/mt_ubench_spdlog.cpp
+ * \author Greg Law
+ * \brief L3: spdlog-multi-threaded logging micro perf-benchmarking
+ * \version 0.1
+ * \date 2024-06-25
+ *
+ * \copyright Copyright (c) 2024
+ *
+ * Usage: program-name [ <number-of-threads> ]
+ * *****************************************************************************
+ */
+#include <stdlib.h>
+#include <pthread.h>
 #include <spdlog/spdlog.h>
 #include <spdlog/sinks/basic_file_sink.h>
-#include <stdlib.h>
 
-#include <pthread.h>
+
+// Micro perf benchmarking workload parameter defaults
+constexpr int L3_PERF_UBM_NTHREADS = 10;
+constexpr int L3_PERF_UBM_NMSGS = (1000 * 1000);
+constexpr int L3_PERF_UBM_LOG_SIZE = (16 * 1024);
```

Comment:
For consistency in workload, define the spdlog-backtrace # entries to be the same as that's being used, by default, for L3 logging.